<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstpage</class>
 <widget class="QWidget" name="firstpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>膳食助手</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.svg</normaloff>icon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="name_label">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>膳食助手</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="slogan_label">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>均衡营养，健康生活。</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="newRecipe_Button">
     <property name="text">
      <string>新建食谱</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Browse_Button">
     <property name="text">
      <string>膳食指南</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
